### ISSUE TITLE
Active Tags Causes Modules to Jump

### ISSUE THREAD

**Issue by zaracay** [SCIInstitute/SCIRun#1058]
Clicking on modules bounce out

**Comment by zaracay** [comment]
Mac version

**Comment by jessdtate** [comment]
not sure what this means.  If limited to tags then it is not as important. I don't know if we will get to tags  IBBM.

**Comment by dcwhite** [comment]
@zaracay @jessdtate Is this still happening?

**Comment by jessdtate** [comment]
Yes.  when you hold down the option button, the modules will jump positions most of the time.

**Comment by dcwhite** [comment]
Need to test with already-jumpy Qt5 network editor.

**Comment by dcwhite** [comment]
**Triage 2026-06:** Active Tags cause modules to jump when clicked — the last comment says this is stale after 240 days with no activity. The tag layer overlay interacts with module widget positioning. If this is still reproducible on current code, the fix is in the tag overlay's mouse event handling in `NetworkEditor`. Suggest testing on current master before investing in a fix.
